src/github.com/getlantern/balancer/dialer.go

```diff
@@ -131,7 +131,7 @@ func (d *dialer) defaultCheck() bool {
 			Dial: d.Dial,
 		},
 	}
-	ok, timedOut, _ := withtimeout.Do(10*time.Second, func() (interface{}, error) {
+	ok, timedOut, _ := withtimeout.Do(60*time.Second, func() (interface{}, error) {
```

Comment:
In general I actually think it's better to only make things constants only if they're used multiple times. In theory making something a constant makes it more descriptive, but to me there's nothing more self-documenting than where something like this is actually used in the code itself. As an illustrations of that, if this were a constant, I can't imagine ever changing it in the future without also looking at this section of code that actually uses it, in which case it just creates more indirection, would take more time, and IMHO is actually less clear and understandable.

That's my view on it, although honestly I actually don't think it's very consequential either way and actually don't care much despite the long description above!